diagnostic.md

```diff
@@ -23,7 +23,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('.listerItem').parents('form');
```

Comment:
Ha! I had never run into parents() before, good find!
In that case, I would probably do `$('li').parents('body').children('form')`. Which is awesome, as it's shorter than the other way. The way you did it wouldn't work because form is a sibling of the ol, so it's never a parent of the li. 
Thanks for showing me that, though!